src/index.html

```diff
@@ -20,8 +21,348 @@
+          <li class="nav__item">
+            <a
+              href="#laptops"
+              data-qa="nav-hover"
+              class="nav__link"
+            >
+              laptops & computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#gadgets"
+              class="nav__link"
+            >
+              gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#tablets"
+              class="nav__link"
+            >
+              tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#photo"
+              class="nav__link"
+            >
+              photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#video"
+              class="nav__link"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- #endregion -->
+    <!-- #region main -->
+    <main>
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="images/imac.jpeg"
+          alt="product's photo"
+          class="card__photo"
+        />
+        <p class="card__describe">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__text">Price:</p>
+          <p class="card__amount">$2,199</p>
+        </div>
+        <a
+          href="#"
+          class="button"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </div>
+      <div class="card">
+        <img
+          src="images/imac.jpeg"
+          alt="product's photo"
+          class="card__photo"
+        />
+        <p class="card__describe">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__text">Price:</p>
+          <p class="card__amount">$2,199</p>
+        </div>
+        <a
+          href="#"
```

Comment:
Consider replacing the '#' in the href attribute with a valid URL or JavaScript function to provide meaningful navigation or action when the 'Buy' button is clicked.